lib/views/frontend/spree/checkout/existing_payment/_paypal_braintree.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr id="<%= dom_id(wallet_payment_source, 'spree') %>" class="<%= cycle('even', 'odd') %>">
+  <td>
+    <%= radio_button_tag "order[wallet_payment_source_id]", wallet_payment_source.id, default, class: "existing-cc-radio" %>
+  </td>
+  <td>
+    <%- if wallet_payment_source.payment_source.credit_card? %>
+      <div><%= wallet_payment_source.payment_source.cardholder_name %></div>
+      <div><%= wallet_payment_source.payment_source.card_type %> - <%= wallet_payment_source.payment_source.display_number %></div>
+      <div><%= wallet_payment_source.payment_source.expiration_month %>/<%= wallet_payment_source.payment_source.expiration_year %></div>
+    <%- elsif wallet_payment_source.payment_source.paypal? %>
+      <div>Paypal</div>
```

Comment:
Should be PayPal (Pal is also capitalized)

Reply:
Oh, Shoot.  Didn't mean to include this here.

Will pull it out. See other pull request to see if we want to pull that in.

Thanks!